kubernetes/tools/devaccount/deploy-dev-account-payload.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash -e
+
+kubectl apply -k .
+
+mkdir -p /home/waggle/.kube
+chown waggle:waggle /home/waggle/.kube
+
+cat <<EOF > /home/waggle/.kube/config
+apiVersion: v1
+clusters:
+contexts:
+current-context:
+kind: Config
+preferences: {}
+EOF
+
+chown waggle:waggle /home/waggle/.kube/config
```

Comment:
it is probably better off to just do a `chown -r` at the end of `/home/waggle/.kube`.

Now if I remember right... when the user runs any commands on the nodes now with `kubectl` they have to do so as `sudo`.  and therefore the `.kube` folder is owned by `root`

see my `waggle` user on W030

```
root@ws-nxcore-000048B02D15BC68:/home/waggle# ls -la
total 48
drwxr-xr-x 7 waggle waggle 4096 Dec  5 02:49 .
drwxr-xr-x 1 root   root   4096 Dec  2 21:09 ..
-rw------- 1 waggle waggle  247 Dec  5 02:49 .bash_history
-rw-r--r-- 1 waggle waggle  220 Apr  4  2018 .bash_logout
-rw-r--r-- 1 waggle waggle 3771 Apr  4  2018 .bashrc
drwx------ 2 waggle waggle 4096 Dec  2 21:16 .cache
drwxr-xr-x 2 waggle waggle 4096 Apr 23  2021 Desktop
drwx------ 3 waggle waggle 4096 Dec  2 21:16 .gnupg
drwxr-x--- 3 root   root   4096 Dec  2 21:17 .kube
-rw-r--r-- 1 waggle waggle  807 Apr  4  2018 .profile
drwxr-xr-x 2 waggle waggle 4096 Dec  2 21:09 .ssh
-rw-r--r-- 1 waggle waggle 2090 Oct 16  2020 .xsessionrc
```

i therefore dont think the files need to be owned by `waggle`

Reply:
The `/home/waggle/.kube` is owned by the Waggle user now.